### DISCUSSION TITLE
Any example of (*Map) Update ??

### DISCUSSION THREAD

**Discussion by YJesus** [cilium/ebpf#790] [Ideas]
There are lot of examples for (*Map) Lookup but any for Update. 

What I want to translate from C to Go is :

map_fd = bpf_object__find_map_fd_by_name(obj->obj, "NameOfMap");

bpf_map_update_elem(map_fd, &key, &value, BPF_ANY);

Thank you

**Comment by ti-mo** [comment]
Hi @YJesus,

There may be no examples of `Map.Update()`, but it's used in many places throughout the codebase. It works virtually the same as `Map.Lookup()`.

See https://sourcegraph.com/github.com/cilium/ebpf@c5668cfdcce4254e8137cadf79757aefaf72e68a/-/blob/map.go?L677:15#tab=references.
